src/components/common/Date/DateBox.js

```diff
@@ -2,72 +2,124 @@ import "./Date.scss";
 import TextBox from "../TextBox/TextBox";
 import { useEffect, useState } from "react";
 
-export default function DateBox({ className, isAlways, handleDate }) {
+export default function DateBox({
+  className,
+  isAlways,
+  handleDate,
+  stateInitialize,
+}) {
   const [date, setDate] = useState({
     year: new Date().getFullYear(),
     month: new Date().getMonth() + 1,
-    day: new Date().getDate()
-  })
+    day: new Date().getDate(),
+  });
+
+  useEffect(() => {
+    const SaveDate = (stateInitialize || "").split("-");
+    setDate({
+      year: SaveDate[0],
+      month: SaveDate[1],
+      day: SaveDate[2],
+    });
+  }, [stateInitialize]);
```

Comment:
```suggestion
  useEffect(() => {
    const splittedDate = (savedDate || "").split("-");
    setDate({
      year: splittedDate[0],
      month: splittedDate[1],
      day: splittedDate[2],
    });
  }, [savedDate]);
```

변수 이름은 카멜케이스
의미가 잘 드러나는 이름을 써주세용